up-and-running/platforms/aws-cloud-formation-template.md

```diff
@@ -0,0 +1,323 @@
+# AWS Cloud formation template.
+
+#### This template is intended to provide a functional, baseline for npmE install on AWS with the following software:
+
+* Ubuntu 16.04 LTS
+* linux kernel 4.4.0-1016-aws
+* node 6.10.2
+* npm 3.10.10
+* npme 4.3.10
+* replicated 2.9.0
+* docker 17.03.1-ce, build c6d412e
+
+While setting up the instance, choose `Production` or `Testing` for the Environment settings.
+This will select `r3.large` or `r3.xlarge` respectively.
+
+| Size |  vCPU/ECU |  GB RAM | GB SSD |
+| ---- | --------- | ------- | ------ |
+| r3.large | 2,6.5 | 15 | 32 |
+| r3.xlarge |4,13 | 30.5 | 80 |
+
+
+#### Key points to note down while setup process.
+
+**Admin Console Password**
+
+The template will require you to set a password for the admin console. Don't lose this. Once you've set it, you'll have to have it to access the admin console and complete the setup.
+
+**Network Security**
+
+This template will allow you to limit incoming connections to SSH, the npmE Admin Panel and the two registry ports (8080, 8081). Each of these defaults to allowing connections from anywhere.
+
+You will have to have at least one EC2 key pair created before using this template to select so that you can SSH into the instance when it's complete.
+
+**Initial Start Time**
+
+Even after the instance starts, it will still take a few minutes for all of the steps to complete. Please be patient as this can take between 5 to 10 minutes. Once the process has completed, navigate your browser to http://<instances public ip>:8800 and follow the setup instructions.
```

Comment:
Looks like I missed an apostrophe: please change the last bit to:

 navigate your browser to `http://<instance's public ip>:8800` and follow the setup instructions.